pangeo_fish/hmm/filter.py

```diff
@@ -43,6 +45,12 @@ def score(emission, predictor, initial_probability, mask=None):
         prediction = predictor.predict(previous, mask=mask)
         updated = prediction * dask.compute(emission[index, ...])[0]
 
+        if np.sum(updated) == 0:
```

Comment:
this computes the sum twice. Can we instead assign a name to the result and use it here and when we append? Something like
```suggestion
        normalization_factor = np.sum(updated)
        if normalization_factor == 0:
```